aws/cloudformation/scylla.yaml.j2

```diff
@@ -61,6 +61,11 @@ Parameters:
     Type: 'AWS::EC2::KeyPair::KeyName'
     ConstraintDescription: must be the name of an existing EC2 KeyPair.
 
+  VpcCidrIp:
+    Description: Set CIDP IP range for VPC
+    Type: String
+    ConstraintDescription: must be valid subnet
```

Comment:
how about doing `172.31.0.0/16` as default ?

Reply:
The requirement (for AWS marketplace i assume) is to configure the VPC range. User must supply it, i am not sure about default , @mailmahee , any input?